src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,13 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
```

Comment:
Name should suggest that `BulldozerProducer` is an implementation of something, so better a name would be: `BulldozerProducerImpl`, same for other implementations.